tests/tests/rap_clm_lake_debug

```diff
@@ -23,3 +23,4 @@ export CCPP_SUITE=FV3_RAP_clm_lake
 export LKM=1
 export IOPT_LAKE=2
 export FRAC_ICE=.true.
+export WRITE_DOPOST=.false.
```

Comment:
@jkbk2004 @ulmononian Can you also incorporate this change in PR #2461? With the recent UPP updates, the inline post is not working for this test anymore.